edx_proctoring/statuses.py

```diff
@@ -288,3 +288,34 @@ def get_onboarding_status_from_attempt_status(cls, status):
         if status:
             return cls.onboarding_statuses.get(status)
         return cls.not_started
+
+class VerificientOnboardingAttemptStatus:
+    """
+    A class for mapping onboarding statuses from Verificient's onboarding status endpoint to
+    internal database values.
+    """
+    no_profile = 'no-profile'
+    approved = 'approved-in-course'
+    other_course_approved = 'approved-in-different-course'
+    rejected = 'rejected'
+    expired = 'expired'
+    pending = 'pending'
+
+    attempt_statuses = {
+        no_profile: None,
```

Comment:
Are we concerned about more granular statuses such as `created`, `ready_to_start` etc.? If the plan is to only use this API endpoint, we will lose that granularity, so is this simple mapping okay right now?

Reply:
I think it's fine and we should actually go further to make it clear that attempt status and onboarding status are two different things. I'd actually suggest renaming this model, and the associated `get_attempt_status_from_api_status` to make that more clear. (removing the word attempt)